docs/usage/setup.md

```diff
@@ -1,14 +1,25 @@
 # Setup
 
-Firstly you need to open the browser at the link <http://localhost:8091> and edit the settings for Zwave, MQTT and the Gateway.
+To configure ZWavejs2Mqtt, you must access it via your web browser at <http://localhost:8091> on the machine on which it was run, or at the IP address of your remote installation on port 8091.
+
+## Quick Start
+
+At the most basic, you must confiugre Z-Wave -> Serial Port and you should:
+
+- Add Network Security Keys - Z-Wave -> Security Keys (S0 Legacy, S2  Unauthenticated, S2 Authenticated, and S2 Access Control)
```

Comment:
```diff
- S2  Unauthenticated
+ S2 Unauthenticated
```

Reply:
We should mention that users should provide **ALL** of those keys, even if they don't need them all at the moment. Also, backup somewhere else in case z2m gets nuked.